trie-db/src/triedbmut.rs

```diff
@@ -1625,11 +1776,4 @@ mod tests {
 		test_comb((0, &a), (1, &b), (1, &[0x01, 0x23, 0x46, 0x78][..]));
 		test_comb((1, &a), (1, &b), (0, &[0x23, 0x46, 0x78][..]));
 	}
-
-	#[test]
-	fn nice_debug_for_node() {
```

Comment:
Removed due to absence of an easy way to have a Layout with a u32 hash.
(could switch back to having Node<H> instead of Node<L>, but switching to Node<L> is something I did a few time in the past so I did not revert those changes, but doable).